ecmascript/regexp-lookbehind.html

```diff
@@ -0,0 +1,207 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<!--
+Copyright (C) 2017 the V8 project authors. All rights reserved.
+This code is governed by the BSD license found in the LICENSE file.
```

Comment:
Hmm, so I wonder about this. Happily everyone here is using the same license, so I don't think there's a real problem. But the test262 copies of these tests don't specifically refer to the V8 copy of the license file (see e.g. [this test](https://github.com/tc39/test262/blob/main/test/built-ins/RegExp/lookBehind/alternations.js)), so I interpret them as referring to the Test262 License file: https://github.com/tc39/test262/blob/main/LICENSE

wpt claims "Copyright © web-platform-tests contributors", so all of this seems to be consistent with that; maybe it's enough to just reproduce the original copyright notice and not link to a different LICENSE file?

Reply:
I wasn't sure, but I guess you're right. Happy to just remove the link then.